sqlite/README.md

```diff
@@ -0,0 +1,364 @@
+7|Facelift|5
+8|Warner 25 Anos|6
+9|Plays Metallica By Four Cellos|7
+10|Audioslave|8
+```
+
+This means that SQLite runs on Unikraft waiting for further instructions.
+
+### Run on QEMU/x86_64
+
+To run the Unikraft image:
+
+```console
+qemu-system-x86_64 \
+  -nographic \
+  -kernel workdir/build/sqlite_qemu-x86_64 \
+  -append "sqlite_qemu-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /chinook.db 'SELECT * FROM Album LIMIT 10'" \
+  -initrd ./initrd.cpio
+```
+
+### Run on QEMU/ARM64
+
+To run the Unikraft image:
+
+```console
+qemu-system-aarch64 \
+  -nographic \
+  -machine virt \
+  -cpu max \
+  -kernel workdir/build/sqlite_qemu-arm64 \
+  -append "sqlite_qemu-arm64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- /chinook.db 'SELECT * FROM Album LIMIT 10'" \
+  -initrd ./initrd.cpio
+```
+
+### Run on Firecracker/x86_64
+
+To run the Unikraft image:
+
+```console
+rm -f firecracker.socket
+firecracker-x86_64 --config-file fc.x86_64.json --api-sock firecracker.socket
+```
+
+The user running the above command must be able to use KVM.
+Typically this means being part of the `kvm` group.
+Otherwise, run the command above as root or prefixed by `sudo`.
+
+### Run on Firecracker/ARM64
+
+To run the Unikraft image:
+
+```console
+rm -f firecracker.socket
+firecracker-aarch64 --config-file fc.arm64.json --api-sock firecracker.socket
+```
+
+The user running the above command must be able to use KVM.
+Typically this means being part of the `kvm` group.
+Otherwise, run the command above as the `root` account or prefixed by `sudo`.
+
+### Run on Xen/x86_64
+
+To run the Unikraft image:
+
+```console
+sudo xl create -c xen.x86_64.cfg
+```
+
+You need use `sudo` or the `root` account to run Xen.
+
+### Run on Xen/ARM64
+
+To run the Unikraft image:
+
+```console
+sudo xl create -c xen.arm64.cfg
+```
+
+You need use `sudo` or the `root` account to run Xen.
+
+## Test
+
+### Close QEMU
+
+To close the QEMU virtual machine, use the `Ctrl+a x` keyboard shortcut;
+that is press the `Ctrl` and `a` keys at the same time and then, separately, press the `x` key.
+
+### Close Firecracker
+
+To close the Firecracker virtual machine, open another console and use the command:
+
+```console
+sudo pkill -f firecracker
+```
+
+### Close Xen
+
+To close the Xen virtual machine, open another console and use the command:
+
+```console
+sudo xl destroy sqlite
+```
+
+## Clean Up
+
+Doing a new configuration, or a new build, may require cleaning up the configuration and build artifacts.
+
+In order to remove the build artifacts, use:
+
+```console
+make clean
+```
+
+In order to remove fetched files also, that is the removal of the `workdir/build/` directory, use:
+
+```console
+make properclean
+```
+
+In order to remove the generated `.config` file as well, use:
+
+```console
+make distclean
+```
+
+## Customize
+
+### Customize the Filesystem Contents
+
+The main way to customize SQLite is to update its configuration in the directory of the filesystem:
```

Comment:
These instructions don't tell me anything. What can I do to update the configuration? What is the `script.sql` file? It it used anywhere?